certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -15,6 +15,20 @@
       "url": "https://www.encodian.com/products/encodian-for-microsoft-power-automate/"
     }
   },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.encodian.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://support.encodian.com/hc/en-gb/articles/360010885513-Privacy-Policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Productivity; Documents; Compliance;"
```

Comment:
@Encodian Please update the value of "Categories" property as per the latest swagger you have uploaded in the portal - It should be `"propertyValue": "Content and Files;Collaboration"`
[Instructions to add Categories in Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#product-or-end-service-metadata)

Reply:
Hey @sriyen-msft updated as requested